### PR TITLE
Unify DialogHolderActivity and PermissionsSupportActivity

### PR DESCRIPTION
**Pull request by DavDo**
MOB 3166

**Jira issue:**
https://glia.atlassian.net/browse/MOB-xxx

**What was solved?**

**Release notes:**

 - [ ] Feature
 - [x] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests updated? Added? Unit, acceptance, snapshots?
 - [ ] Logging for future troubleshooting of client issues added?

**Screenshots:**

### CODE REVIEW

File: widgetssdk/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -68,14 +68,9 @@
...
             android:launchMode="singleTask"
             android:theme="@style/Application.Glia.Call.Activity" />
-        <activity
-            android:name=".permissions.PermissionsSupportActivity"
-            android:launchMode="singleTask"
-            android:noHistory="true"
-            android:theme="@style/Application.Glia.Translucent.Activity" />
 
         <activity
-            android:name=".view.dialog.holder.DialogHolderActivity"
```
Comment (andrews-moc) [inline widgetssdk/src/main/AndroidManifest.xml:78]: To be honest, I like the `DialogHolderActivity` name more. To me, `GhostActivity` is associated with some spy activity and may sound dangerous for integrators.

Comment (DavDo) [reply widgetssdk/src/main/AndroidManifest.xml:78]: Makes sense. Will change it back

Comment (DavDo) [reply widgetssdk/src/main/AndroidManifest.xml:78]: fixed

**Review by AndriiHorishniiMOC — APPROVED** [review]
LGTM!

**Review by andrews-moc — APPROVED** [review]

**Pull request merged**